src/routes/buildManifestObject.js

```diff
@@ -79,11 +88,29 @@ export function buildManifestObject() {
       type: 'okapi',
       records: 'items',
       resultOffset: '%{resultOffset}',
-      perRequest: 100,
       throwErrors: false,
       path: (queryParams) => {
-        const queryValue = get(queryParams, 'query', '');
-        if (queryParams.qindex === browseModeOptions.CALL_NUMBERS) return `browse/call-numbers/instances?expandAll=true&query=callNumber>=${queryValue} or callNumber<${queryValue}&precedingRecordsCount=5&`;
+        const prevNextReg = /^(callNumber [<|>])/ig;
+        const query = get(queryParams, 'query', '');
+
+        if (prevNextReg.test(query)) {
+          return `browse/call-numbers/instances?${new URLSearchParams({
+            expandAll: true,
+            highlightMatch: false,
+            query,
+            precedingRecordsCount: 25,
+            limit: 25
+          })}&`;
+        }
+
+        if (queryParams.qindex === browseModeOptions.CALL_NUMBERS) {
+          return `browse/call-numbers/instances?${new URLSearchParams({
+            expandAll: true,
+            query: `callNumber>=${query} or callNumber<${query}`,
+            limit: 25
+          })}&`;
```

Comment:
it might be better to mutate this query from the SearchIndex selection within InstanceList... (within `onChangeSearchIndex`)  and actually move this `path` function to the `path` key of `records`, with it returning the default `inventory/instances` value.

This way you can avoid having to change the contents of `parentResources` within InstancesRoute.

Reply:
Redacted the suggestion here as there isn't any way in `stripes-connect` for the `records` to be functional... but at least those calls with the same `records` key can be consolidated with a functional path (as you have it)

Reply:
Thanks for comments, I will think about it in future PR related to nex-prev behavior for subjects